stdlib/stringLabels.mli

```diff
@@ -273,6 +273,17 @@ val equal: t -> t -> bool
 (** The equal function for strings.
     @since 4.05.0 *)
 
+val unseeded_hash : t -> int
+(** An unseeded hash function for strings, with the same output value as
+    {!Hashtbl.hash}.
+    @since 4.10.0 *)
+
+val hash : seed:int -> t -> int
+(** A seeded hash function for strings, with the same output value as
+    {!Hashtbl.seeded_hash}. This function allows the module [StringLabels]
+    to be passed as argument to the functor {!Hashtbl.MakeSeeded}.
+    @since 4.10.0 *)
```

Comment:
I don't think you should put a label here if you want to be able to pass it to {!Hashtbl.MakeSeeded}. Maybe if you wanted to pass it to [MoreLabels.Hashtbl.MakeSeeded] but no, since in that one  `module type SeededHashedType = Hashtbl.SeededHashedType`. 

I think this shows once more why the whole labelled `stdlib` is not really sunstainable design wise and should rather [die](https://github.com/ocaml/ocaml/issues/7400).